app/src/main/res/values-nl/strings.xml

```diff
@@ -85,7 +85,7 @@
     <string name="yes_go_back">Ja, ga terug!</string>
 
     <!-- Kies verificatie methode -->
-    <string name="select_verification_method">Selecteer een verificatie methode</string>
+    <string name="select_verification_method">Hoe wil je verbinden?</string>
```

Comment:
Ik denk niet dat dit de juiste vertaling is:

Mobiel
> "Hoe wil je verbinden?"

Gebruiker
> Via Bluetooth, daar heb ik net voor gekozen 😕 


Misschien iets als "Hoe wil je de verbinding beveiligen?", of iets dergelijks?